Prace_domowe/Praca_domowa4/Grupa4/Patryk_Tomaszewski/PD4.ipynb

```diff
@@ -0,0 +1,1112 @@
+{
```

Comment:
<p>Hmm, jakby to powiedzieć... w zbiorze dalex targetem jest m2.prcie</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/mini-pw/2021L-WUM/pull/228/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='9'/>

Reply:
<p>Poprawione</p>